plugins/modules/helm_get.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+
+module: helm_get
+
+short_description: Download extended information of a named release
```

Comment:
This module seems to have a lot of overlap in purpose with `helm_info`. Can we augment the info module with this information instead of creating a new module?

Reply:
module removed and `helm_info` updated with new info